views/index.edge

```diff
@@ -0,0 +1,136 @@
+@layout('layouts.app')
+
+@section('content')
+
+<!-- Main Content -->
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <h1>Raven infrastructure status</h1>
+    </div>
+  </div>
+  <div class="row mt-5">
+    <div class="col-lg-4">
+      <h3>Community</h3>
```

Comment:
This look like something you can template instead of repeating it three times, for each category.